pkg/controller/metrics/metrics.go

```diff
@@ -155,6 +155,115 @@ var (
 	// mapMetricToDurationGauges is a map of optional durationMetrics of type Gauge to their specific duration, if
 	// mentioned
 	mapMetricToDurationGauges map[*prometheus.GaugeVec]time.Duration
+
+	// Metrics reported by ClusterDeployment controller
+
+	MetricInstallJobDuration = prometheus.NewHistogramVec(
```

Comment:
You might notice, I changed the type of some of the metrics, especially ones declared as Histogram or HistogramOpts, for consistency.